packages/components/src/lib/SongCard.tsx

```diff
@@ -1,225 +1,221 @@
-import { type KeyboardEvent, useCallback, useEffect, useState } from "react";
-import { IconButton, Stack, Typography, useTheme } from "@mui/material";
+import { type KeyboardEvent, MouseEvent, useCallback } from "react";
+import { Box, IconButton, Stack, Typography, useTheme } from "@mui/material";
 import { PlayArrow, Stop } from "@mui/icons-material";
 import { bgImage } from "@newm-web/assets";
-import {
-  getImageSrc,
-  resizeCloudinaryImage,
-  useWindowDimensions,
-} from "@newm-web/utils";
-import { SongCardSkeleton } from "@newm-web/elements";
+import { getImageSrc, resizeCloudinaryImage } from "@newm-web/utils";
+import { Clickable, SongCardSkeleton } from "@newm-web/elements";
 
 interface SongCardProps {
-  coverArtUrl?: string;
-  isPlayable?: boolean;
-  isPlaying?: boolean;
-  onPlayPauseClick?: () => void;
-  onPriceClick?: () => void;
-  onSubtitleClick?: () => void;
-  onTitleClick?: () => void;
-  price?: string;
-  size?: "Small" | "Large";
-  subtitle?: string;
-  title?: string;
+  readonly coverArtUrl?: string;
+  readonly isLoading?: boolean;
+  readonly isPlayable?: boolean;
+  readonly isPlaying?: boolean;
+  readonly onCardClick?: () => void;
+  readonly onPlayPauseClick?: () => void;
+  readonly onPriceClick?: () => void;
+  readonly onSubtitleClick?: () => void;
+  readonly price?: string;
+  readonly subtitle?: string;
+  readonly title?: string;
 }
 
-// create a record for the song card size, small is 260 x 260, large is 400 x 400
-const songCardSize: Record<string, number> = {
-  Large: 400,
-  Small: 260,
-};
-
 export const SongCard = ({
   coverArtUrl,
   title,
   isPlayable,
   isPlaying,
-  onTitleClick,
+  onCardClick,
   onPlayPauseClick,
   onPriceClick,
   onSubtitleClick,
   price,
   subtitle,
-  size = "Small",
+  isLoading = true,
 }: SongCardProps) => {
   const theme = useTheme();
-  const windowWidth = useWindowDimensions()?.width;
-
-  const [isWidthAboveMd, setIsWidthAboveMd] = useState(true);
-  const [isLoading, setIsLoading] = useState(true);
-
-  useEffect(() => {
-    if (windowWidth !== undefined) {
-      setIsLoading(false);
-    }
-  }, [windowWidth]);
 
-  useEffect(() => {
-    setIsWidthAboveMd(
-      !!(windowWidth && windowWidth > theme.breakpoints.values.md)
-    );
-  }, [theme.breakpoints.values.md, windowWidth]);
+  const handleCardClick = (event: MouseEvent | KeyboardEvent) => {
+    event.preventDefault();
+    event.stopPropagation();
+    onCardClick?.();
+  };
 
-  const handlePriceClick = () => {
-    onPriceClick?.();
+  const handlePlayPauseClick = (event: MouseEvent | KeyboardEvent) => {
```

Comment:
After reviewing it again, once I add `onPlayPauseClick={ () => {} }` to the `SongCard`  component in `Songs` , the `stopPropagation` functions correctly. Was just missing the event listener on the play/pause button to trigger instead of only having the card trigger.